src/main.c

```diff
@@ -294,7 +296,7 @@ static char *strtok_r(char *str, const char *delim) {
     return NULL;
 }
 
-static int hide_file(const char *path_name) {
+int hide_file(const char *path_name) {
     int retval;
 	char *dir_path;
 	char *file_name;
```

Comment:
This is not a new change so i cant comment on it directly...
on line 309, `strtok_r` is the reentrant version of strtok, not the reverse. Which means it probably doesn't do what you expect, and also i believe you are missing an arg

Reply:
I implemented `strtok_r` by myself based on the behavior of `strtok` (since they are not provided by the kernel), and didn't realize it behaves differently from `strtok_r` from the standard library.
I'll read about the real `strtok_r` and either change my function (if it fits here) or rename it to something unique.

Reply:
Ok I understand the difference.
I think I'll rename my function, and simplify it as well. (no real need to accept multiple delimiter characters, getting only one `char` will allow me to use `strrchr`).

Out of curiosity, is it considered ok to use `strtok` normally? to me having an internal state seems kinda fishy.